.github/workflows/deploy-it.yml

```
@@ -0,0 +1,83 @@
+# This workflow registers the notifications image for deployment to IT, and can be manually triggered,
+# or automatically triggered by a push to the main branch
+
+name: Build and deploy to IT
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    env:
+      NAMESPACE: 'ehealthid-relying-party'
+      DOCKER_REPO: 'europe-docker.pkg.dev/oviva-pkg/ovi/'
+      IMAGE_NAME_BACKEND: 'ehealthid-relying-party'
```

Namespace is not used at all inside the file. Repo needs to be changed to the GitHub Container Registry.
```suggestion
      DOCKER_REPO: 'ghcr.io/oviva-ag/'
      IMAGE_NAME_BACKEND: 'ehealthid-relying-party'
```